IPs/RSKIP197.md

```diff
@@ -0,0 +1,78 @@
+# Error Handling for Precompiled Contracts
+
+|RSKIP          |197           |
+| :------------ |:-------------|
+|**Title**      |Error Handling for Precompiled Contracts |
+|**Created**    |15-12-2020 |
+|**Author**     |FJ |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Accepted |
+
+## Abstract
+
+This RSKIP specifies a new expected behavior for a precompiled contract call: any expected failed call pushes a zero into the stack and reverts the state before the execution.
```

Comment:
What is an "expected failed call" ?

Reply:
this part seems unclear to me also, I was trying to point those scenarios where you expect the precompiled to revert the transaction